content/en/security/synthetics.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Synthetics Monitoring Security
+kind: documentation
+aliases:
+    - /synthetics/security/
+further_reading:
+- link: "/security/"
+  tag: "Documentation"
+  text: "Review the main categories of data submitted to Datadog"
+---
+
+<div class="alert alert-info">This page is about the security of Datadog; if you're looking for the Security Monitoring product, see the <a href="/security_monitoring" target="_blank">Security Monitoring section</a>.</div>
+
+This article is part of a [series on data security][1].
+
+The [Synthetics product][2] allows you to proactively monitor how your systems and applications are performing using simulated requests and business transactions. Synthetic tests can be initiated from all around the globe, from either managed or private locations.  
+This article describes the main security guarantees available to users leveraging the Synthetics product, for both location types.
+
+# Encryption in managed locations 
+
+## Test configurations and variables
+
+* **Transport**: Asymmetric encryption - RSA (4096-bit key). All requests are signed using Datadog Signature v1 (based on the same signing process as [AWS Signature v4][3]), ensuring both authentication and integrity. 
+* **Storage**: Symmetric encryption - AES-GCM (256-bit key).
+
+## Test results
+
+* **Transport**: Asymmetric encryption - RSA (4096-bit key). All requests are signed using Datadog Signature v1 (based on the same signing process as [AWS Signature v4][3]), ensuring both authentication and integrity.
+* **Storage**: Sensitive parts (response headers and body) of test results are stored encrypted with an asymmetric encryption - RSA (4096-bit key) and decrypted on-the-fly when test results are fetched.
+
+## Artifacts
+
+Artifacts are browser test screenshots, snapshots, errors, and resources.
+
+{{< site-region region="us" >}}
+
+* **Transport**: Encryption in transit using [AWS Signature Version 4 for S3][1].
+
+[1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/network-isolation.html
```

Comment:
Can you double-check that this is the right target URL? It mentions AWS Signature V4 for S3, but doesn't explain it.

Reply:
Yeah so we used that page as a "proof" that s3 uses sigv4. It also mentions TLS, which is nice. We could use https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html instead which is more detailed for instance but that's the API doc so it feels weird. This one https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html goes into more details about sigv4 but does not mention s3. So let's keep that one 👍